patches/third_party-lottie-lottie_worker.js.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/third_party/lottie/lottie_worker.js b/third_party/lottie/lottie_worker.js
+index adb3ff4793654284dc3e5790a140dfd97e0cc8a6..61e216f8cbf65f22e94688cd267446a51529ac5a 100644
+--- a/third_party/lottie/lottie_worker.js
++++ b/third_party/lottie/lottie_worker.js
+@@ -11947,6 +11947,7 @@ var events = {
+   PLAYING: 'playing',  // Send when the animation started playing.
+   PAUSED: 'paused',  // Sent when the animation has paused.
+   STOPPED: 'stopped',  // Sent when the animation has stopped.
++  COMPLETE: 'complete' // Sent when animation is completed.
+ };
+ 
+ /**
+@@ -11987,6 +11988,7 @@ function sendPauseEvent() {
+   postMessage({name: events.PAUSED});
+ }
+ 
++
+ /**
+  * Informs the parent thread that the animation is stopped.
+  */
+@@ -11994,6 +11996,13 @@ function sendStopEvent() {
+   postMessage({name: events.STOPPED});
+ }
+ 
++/**
++ * Informs the parent thread that the animation is complete.
++ */
++ function setCompleteEvent() {
++  postMessage({name: events.COMPLETE});
++}
++
+ /**
+  * Informs the parent thread that the animation has finished initializing.
+  */
+@@ -12033,6 +12042,7 @@ function initAnimation(animationData, initParams, canvas) {
+     }
+   });
+ 
++  currentAnimation.addEventListener("complete", setCompleteEvent);
```

Comment:
If you want to change this whole patch to a 1-liner then perhaps you can try:

```js
currentAnimation.addEventListener("complete", () => postMessage({ name: 'COMPLETE' });
```

Reply:
patch updated